{{cookiecutter.app_name}}/Dockerfile

```diff
@@ -0,0 +1,17 @@
+FROM python:3.6
+
+RUN mkdir -p /usr/src/app
+WORKDIR /usr/src/app
+
+COPY requirements.txt  /usr/src/app/
+
+RUN pip install --no-cache-dir -r requirements.txt
+
+COPY . /usr/src/app
+
+EXPOSE 8000
+
+ENV PYTHONUNBUFFERED=1
+ENV NEW_RELIC_CONFIG_FILE=newrelic.ini
```

Comment:
THIS GUY!!! https://github.com/ButterflyNetwork/olympus-storage/blob/247eecf7779efd1035d9762d28fbecba5a3fab3a/Dockerfile#L24-L25